src/idea.js

```diff
@@ -6,42 +6,44 @@ class Idea  {
     this.star = star;
     this.comments = [];
   }
-  saveToStorage(idea) {
+  saveToStorage() {
```

Comment:
Both functions get retrieved ideas out of local storage, add an idea to the retrieved ideas, and push all ideas back to local storage. The old function took an idea as an argument.
New function goes through a loop, matches up ideas to find an idea and then is the same. 
The other way is way more straightforward and less confusing. I'm not sure why this change is better.